pages/common/lb.md

```diff
@@ -0,0 +1,28 @@
+# lb
+
+> A blog management tool.
+> More information: <https://github.com/LukeSmithxyz/lb>.
+
+- Make a new draft:
+
+`./lb new`
+
+- Edit a draft:
+
+`./lb edit`
+
+- Delete a draft:
+
+`./lb trash`
+
+- Publish a draft:
+
+`./lb publish`
+
+- Delete a published post:
```

Comment:
Here again, which post wil this delete?

Reply:
All subcommands except `new` use the `listandReturn` function, which displays the prompt I described for `edit`, in this case:
```
1 article

Pick an entry by number to trash, or press Ctrl-C to cancel.
```

Reply:
When deleting something users may be a bit more causious about executing that command if they don't know exactly what will happen. I can't come up with a good formulation for the description, so how about adding ``> All subcommands except `new` will list all blog posts to select from`` to the command description?

Reply:
Thought about this for...a while and, while I don't like it, I can't come up with a better solution, except a better _formulation_:

Drafts and posts to operate on are selected interactively when running the commands.

Reply:
Sounds pretty good. I like it.